db/migrate/20220130022920_change_address_column_in_users.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeAddressColumnInUsers < ActiveRecord::Migration[7.0]
+  def change
+    change_column :users, :address, :text
```

Comment:
Instead of adding a new migration for this, go back and modify the previous [one](https://github.com/guty/reporte_ciudadano/pull/14/files#diff-6b25df97451709f53e3705a7ebb8e1cce2aad67b137465bd7553d6c1b224e541R6)

Reply:
Going back means I have to make a rollback and run migrations again?